client/src/components/Badge/Badge.js

```diff
@@ -23,13 +23,13 @@ class Badge extends PureComponent {
       return null;
     }
 
-    const invertedClass = inverted ? `badge-${status}--inverted` : '';
+    const colourClass = inverted ? `text-bg-${status}--inverted` : `text-bg-${status}`;
```

Comment:
https://getbootstrap.com/docs/5.0/migration/#badges

> Dropped all `.badge-*` color classes for background utilities (e.g., use `.bg-primary` instead of `.badge-primary`).

Note that as of bootstrap 5.2 the `text-bg-*` is available which is what I opted for here. See https://getbootstrap.com/docs/5.2/helpers/color-background/